Merging existing DC docs into 0.1 release branch (#1322)
corda/corda-docs-portal: content/en/tools/digital-currencies/digital-currencies-glossary.md

 
date: '2023-03-21'
lastmod: '2023-03-21'
section_menu: tools
menu:
  tools:
    name: "Digital Currencies Glossary"
    weight: 2000
    parent: digital-currencies
    identifier: digital-currencies-roles-glossary
    description: "Digital Currencies documentation glossary."
title: "Digital Currencies Glossary"
 

> ** Central bank digital currency (CBDC)** A central bank-issued digital money, denominated in a national unit of account, which is backed by the full faith and credit of the issuing central bank.

Reviewer: ```suggestion
** Central Bank Digital Currency (CBDC)** A central bank-issued digital money, denominated in a national unit of account, which is backed by the full faith and credit of the issuing central bank.
```

Author: Done

> **Defining** Creating a token definition, which specifies the properties that a token of that type may haves. Defining a token is a similar process for CBDCs and for commerical bank-issued stablecoins; however, the approval process may differ.

Reviewer: ```suggestion
**Defining** Creating a token definition, which specifies the properties that a token of that type may have. Defining a token is a similar process for CBDCs and for commercial bank-issued stablecoins; however, the approval process may differ.
```

> **Defining** Creating a token definition, which specifies the properties that a token of that type may haves. Defining a token is a similar process for CBDCs and for commerical bank-issued stablecoins; however, the approval process may differ.

Reviewer 2: ```suggestion
**Defining** Creating a token definition, which specifies the properties that a token of that type may haves. Defining a token is a similar process for CBDCs and for commercial bank-issued stablecoins; however, the approval process may differ.
```

Author: Done

> **Issuing party** The network party 

Reviewer: ```suggestion
**Issuing party** The network party.
```

Author: Done.

**Redeeming** When a commercial bank or central bank removes a token, which it defined, from a transacting entity or a network.

**Stablecoins** A digital money, issued by an officially chartered organization, such as a bank, that can be backed by fiat, a basket of assets, or faith in the issuer's balance sheet.

> **Token Operator**

Reviewer: Token Operator definition missing

Author: Removed this entry.

> **Token Owner** Also Token Issuing Entity

Reviewer: ```suggestion
**Token Owner** Also Token Issuing Entity.
```

> **Token Definition Authority** The role which must approve  any token definitions. There can be multiple users across the entity which reside in different dept. Their function is to review the token definition presented and approve or reject. Central Bank: Board of Governors. Commercial Bank: Head of Legal Services or Head of Regulation/Compliance.

Reviewer: ```suggestion
**Token Definition Authority** The role which must approve any token definitions. There can be multiple users across the entity which reside in different departments. Their function is to review the token definition presented and approve or reject. Central Bank: Board of Governors. Commercial Bank: Head of Legal Services or Head of Regulation/Compliance.
```

> **Token Defining Entity** Token Defining Entity (TDE) – will author and intiate the token definition. TDE can define multiple token definitions. Central Bank: Currency vault management, Commercial bank : Head of Treasury

Reviewer: ```suggestion
**Token Defining Entity (TDE)** It will author and intiate the token definition. TDE can define multiple token definitions. Central Bank: Currency vault management, Commercial bank : Head of Treasury.
```

Author: Done.

> **Token Defining Entity** Token Defining Entity (TDE) – will author and intiate the token definition. TDE can define multiple token definitions. Central Bank: Currency vault management, Commercial bank : Head of Treasury

Reviewer 2: ```suggestion
**Token Defining Entity** Token Defining Entity (TDE) – will author and initiate the token definition. TDE can define multiple token definitions. Central Bank: Currency vault management, Commercial bank : Head of Treasury
```

Author: Done

**Token Definition Owner** Also Token Definition Authority. 

> **Transacting Entity** All transacting entities on the network that have tokens issued them can actively settle assets on the network in real time between each other.

Reviewer: ```suggestion
**Transacting Entity** All transacting entities on the network that have tokens issued can actively settle assets on the network in real time between each other.
```

Author: Done
